olxcleaner/loader/xml_exceptions.py

```diff
@@ -69,7 +69,7 @@ class InvalidPointer(CourseError):
 
     def __init__(self, filename, **kwargs):
         super().__init__(filename)
-        self._description = f"The {kwargs['edxobj']} tag looks like it is an invalid pointer tag"
+        self._description = f"The {kwargs['edxobj']} tag has already been pointed to and is not allowed to point to another file."
```

Comment:
This reads like the error description of PointerAlreadyPointedAt? I think they need to be swapped?

Reply:
maybe something like `can not be empty and point to another file?`?